Lombiq.HelpfulLibraries.LinqToDb/Readme.md

```diff
@@ -0,0 +1,56 @@
+# LinqToDb - Lombiq HelpfulLibraries for Orchard Core
+
+
+
+## About
+
+With the help of this module you can write LINQ expressions and run it with an `ISession` extension method to query from the DB instead of writing plain SQL queries.
+
+
+## Documentation
+
+Use the `LinqQueryAsync` ISession extension for running LINQ syntax based DB queries.
+```csharp 
+LinqQueryAsync<TResult>(this ISession session,Func<ITableAccessor, IQueryable> query)
+```
+
+### Sample controller
+
+```csharp
+public class LinqToDbSamplesController
+    {

```

Comment:
Fölösleges behúzás, lásd itt is: https://github.com/Lombiq/Helpful-Libraries/blob/6235ff70007e401d1ebf10c42998ff84d858ee2c/Lombiq.HelpfulLibraries.LinqToDb/Readme.md